stdlib/public/stubs/CMakeLists.txt

```diff
@@ -18,8 +18,10 @@ set(LLVM_OPTIONAL_SOURCES
     ${swift_stubs_objc_sources}
     ${swift_stubs_unicode_normalization_sources})
 
-find_package(ICU REQUIRED COMPONENTS uc)
-set(ICU_UC_LIBRARY "")
+if(NOT CMAKE_SYSTEM_NAME STREQUAL "Darwin")
```

Comment:
Hm. That's not 100% correct either, since it's testing the host platform rather than the target one. @modocache, did we ever come up with a good answer for this?